diktat-rules/src/main/resources/rules-config.json

```diff
@@ -342,5 +342,10 @@
     "name": "ENUMS_SEPARATED",
     "enabled": true,
     "configuration": {}
+  },
+  {
```

Comment:
will need to merge conflicts

Reply:
also if you will add your rule to diktat-analysis.yml - please add comments of what does your rule do